content/md/first-steps/manage-multidimensional-data-in-a-table.md

```diff
@@ -0,0 +1,143 @@
+---
+id: manage-multidimensional-data-in-a-table
+themes: catalog-settings
+title: Manage multidimensional data in a **table**
+popular: true
+ee-only: false
+related:
+---
+
+# Overview
+
+Sometimes, it can be useful to present product information in a table, in order to **ease the enrichment**, **improve the readability** of the product sheet, and/or because it is **required by law** (for instance, to present the nutritional values of a product in the food industry).
+
+These are the reasons why we've decided to create the **Table attribute** type.
+
+:::info
+This attribute type is available in the _Growth Edition_ and in the _Enterprise Edition_.
+:::
+
+# How to create a Table attribute within Akeneo?
+
+As for the other attribute types, you can create a Table by clicking on `Settings`, `Attributes`, `Create attribute`. Then, click on `Table` and choose among our **templates**.
+
+:::info
+These templates are here to guide you. They are only **presentation suggestions** and can be modified if they don't match your exact needs. If you prefer to create your table from scratch, then choose the `Empty table` template.
+:::
+
+Once the template is selected, a pop-in appears, asking you to choose a `code`, a `label`, and sometimes a `standard`, depending on the template you just chose.
+
+>image
+
+You'll discover a new tab called `Table structure`: this is where you can define your columns.
+
+>image
+
+Except for the `Empty table` template, some columns are already created for you. You are free to add new ones, simply by clicking on `Add new columns`.  
+For each new column, you must define a code, a label, and the type (`select`, `number`, `text` or `yes/no`). The column label can be translated in your catalog locales.
+You can also **change the order** of the columns, by clicking on the burger button.
+
+>image
+
+Finally, you are also free to **delete** each column, by clicking on the trashcan (appearing when hovering the raw).
+
+:::tips
+Let's imagine you are selling foods worldwide, and you need to add the nutritional values of your products. You know that these nutritional tables are specific to a country or a continent (their format are different) and must obey the law. Then you'll need one nutritional table per geographical area. For instance: one for your US market, one for your European market, and a last one for the UK market. In this case, we advise you to create 3 nutritional tables, with the `locale specific` parameter, in order to avoid having empty tables in some locales.
```

Comment:
```suggestion
Imagine you sell food products worldwide, and you need to add the nutritional values of your products. You know that nutritional tables formats will differ between countries or continents, and that they are regulated by national laws. Therefore, you'll need one nutritional table per geographical area: one for your American market, one for your European market, and a last one for the UK market. In this case, we recommend you to create 3 nutritional tables, with the `locale specific` parameter, in order to avoid having empty tables in some locales.
```

Reply:
Here, I would add a title to say you are actually telling them they should make their TA localizable